worlds/dark_souls_3/docs/locations_en.md

```diff
@@ -466,7 +466,7 @@ ladder which is part of Consumed King's Garden.
   * "Lower" is the bottom floor that you enter onto from the plaza.
   * "Upper" is the top floor with the door to the main hall.
   * "Mid" is the middle floor accessible by climbing a ladder from lower or
-    going down stairs from upper.
+    going downstairs from upper.
```

Comment:
"down stairs" is correct here, since it refers to going down a specific set of stairs (a shortening of "down the stairs") rather than going to the lower floor.